_slides/05-montseacciona.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```

Comment:
Woul be good to remove latest '!' character

Reply:
ok make sense